qml/pages/SettingsPage.qml

```diff
@@ -64,6 +64,40 @@ Page {
                 text: qsTrId("chum-settings-testing")
                 onClicked: Chum.repoTesting = !Chum.repoTesting;
             }
+
+            SectionHeader {text: qsTrId("chum-settings-advanced")}
+
+            Label {
+                anchors {
+                    left: parent.left
+                    leftMargin: Theme.horizontalPageMargin
+                    right: parent.right
+                    rightMargin: Theme.horizontalPageMargin
+                }
+                color: Theme.highlightColor
+                text: qsTrId("chum-settings-override-release")
+            }
+
+            TextField {
+                id: txtRelease
+                anchors {
+                    left: parent.left
+                    leftMargin: Theme.horizontalPageMargin
+                    right: parent.right
+                    rightMargin: Theme.horizontalPageMargin
+                }
```

Comment:
I suggest to add `description` and explain that only version string needs to be added. Maybe having an example would help, such as 4.0.1.24. Right now it is not clear whether I need to specify full URL or just version. Also, would be good to mention that this field has to be cleared for use of automatic version setup.

Maybe we should add `placeholderText` with short example as well?